Jenkinsfile

```diff
@@ -2,7 +2,7 @@ pipeline {
     agent none
     options {
         ansiColor("xterm")
-        timeout(time: 6, unit: "HOURS")
+        timeout(time: 4, unit: "HOURS")
```

Comment:
seems to be other changes mixed in here that are not related to the version mismatch check?

Reply:
yes, the builds now take under 3h so adapted the timeout